runtime/src/commonTest/kotlin/pbandk/testpb/test_messages_proto3.kt

```diff
@@ -1789,6 +1790,7 @@ data class TestAllTypesProto3(
                     )
                 }
                 pbandk.MessageDescriptor(
+                    fullName = "protobuf_test_messages.proto3.TestAllTypesProto3.NestedMessage",
```

Comment:
Same comment as above, shouldn't it be pbandk.testpb.TestAllTypesProto3.NestedMessage